nekoyume/Assets/_Scripts/UI/Widget/WorldMap.cs

```diff
@@ -145,7 +140,7 @@ public override void Initialize()
 
         #endregion
 
-        public void Show(WorldInformation worldInformation)
+        public void Show(WorldInformation worldInformation, bool isEventDungeon = false)
```

Comment:
How about this? The `stageType` without `default` value.
```suggestion
        public void Show(WorldInformation worldInformation, StageType stageType)
```